lucene/core/src/java/org/apache/lucene/index/FieldInfo.java

```diff
@@ -92,6 +97,8 @@ public FieldInfo(
       int vectorDimension,
       VectorEncoding vectorEncoding,
       VectorSimilarityFunction vectorSimilarityFunction,
+      boolean isTensor,
+      TensorSimilarityFunction.Aggregation tensorAggregate,
```

Comment:
naive question: is `isTensor` possible/meaningful without `tensorAggregate` and vice versa? and/or could a special `TensorSimilarityFunction.Aggregation.NONE` mean no-tensors?
```suggestion
      TensorSimilarityFunction.Aggregation tensorAggregate,
```

Reply:
Good point, we could probably skip the isTensor flag by using `TensorSimilarityFunction.Aggregation.NONE`.

Reply:
added https://github.com/apache/lucene/pull/13525/commits/b2b95cdb31c7deb73c8ab5a70e0077772bef4414 to give that a go